transform/mattermost-analytics/models/intermediate/product/nps/int_nps_server_version_spined.sql

```diff
@@ -0,0 +1,32 @@
+with first_server_version as 
+(
+    select 
+        server_id
+        , min(event_date) min_event_date
+    from {{ ref('int_nps_score') }}
+        group by all
+)
+, spined as (
+    select 
+        date_day::date as activity_date
+        , server_id as server_id
+    from first_server_version fsd
+    left join {{ ref('telemetry_days') }} td 
+        on date_day::date >= min_event_date
+) 
+, server_version_cte as (
+    select 
+        sp.activity_date
+        , sp.server_id
+        , COALESCE(nps_score.server_version, FIRST_VALUE(nps_score.server_version IGNORE NULLS) OVER (
```

Comment:
Why is a coalesce needed here? 3/5 it's not needed.

Reply:
Yes, this is not required, will remove it.